### ISSUE TITLE
PDF zeigt erste Zeile des Abstract in Fettdruck, zweite Zeile aber nicht

### ISSUE THREAD

**Issue by mkarg** [ijug-ev/javaaktuell-asciidoc-template#1]
Wenn man einen mehrzeiligen Abstract verwendet, erscheint dessen erste Zeile im PDF in Fettdruck, während seine Folgezeilen nicht fettgedruckt sind.

**Comment by mkarg** [comment]
@McPringle WDYT?

**Comment by McPringle** [comment]
Ja, das PDF ist noch nicht perfekt. Das hat im Moment allerdings auch keine Prio, da die Redaktionsleitung mit der Word-Datei arbeitet. Die muss zuerst brauchbar gemacht werden.
